### PR TITLE
Merge develop to bridge

### PR DESCRIPTION
**Pull request by wer1st**

### CODE REVIEW

File: pallets/demeter-farming-platform/src/tests.rs [hunk 1]
```diff
@@ -2394,17 +2394,6 @@ mod tests {
                     deposit_fee,
                 )
             );
-
-            let pool_infos =
```
Comment (vovac12) [inline pallets/demeter-farming-platform/src/tests.rs:2398]: I think it should not be removed. It's presented in develop branch
https://github.com/sora-xor/sora2-network/blob/e469569e25150f244e6bb11d21a801b889e0d960/pallets/demeter-farming-platform/src/tests.rs#L2368

**Review by N1ghtStorm — APPROVED** [review]

**Review by vovac12 — APPROVED** [review]

File: Jenkinsfile [hunk 2]
```diff
@@ -21,7 +21,7 @@ String contractsEnvFile       = 'env.template'
 String solcVersion            = '0.8.14'
 String nodeVersion            = '14.16.1'
 String gitHubUser             = 'sorabot'
-String gitHubRepo             = 'github.com/soramitsu/sora2-substrate.git'
+String gitHubRepo             = 'github.com/soramitsu/sora2-network.git'
```
Comment (bragov4ik) [inline Jenkinsfile:24]:
```suggestion
String gitHubRepo             = 'github.com/sora-xor/sora2-network.git'
```

File: README.md [hunk 3]
```diff
@@ -54,6 +54,10 @@ cargo build --release --features private-net
 ./run_script.sh -d -w -r
 ```
 access running network via polkadot.js/apps (select Development -> Local Node, e.g. [link](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/explorer))
+#### On macOS
+macOS is shipped with the BSD version of `getopt`. If the script behaves incorrectly and does not parse arguments,  
+make sure you have installed and set as active the GNU versions of `awk` and `getopt`.
```
Comment (bragov4ik) [inline README.md:59]:
```suggestion
make sure you have installed and set as active the GNU versions of `awk` (or `gawk`) and `getopt`.
```

**Review by bragov4ik — APPROVED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (wer1st) [reply Jenkinsfile:24]: Yes, but it is just merge PR.
I think it is better to create another PR to develop branch with your changes

Comment (wer1st) [reply Jenkinsfile:24]: https://github.com/sora-xor/sora2-network/pull/304

**Pull request merged**